cypress/e2e/sign_in_page.spec.js

```diff
@@ -0,0 +1,86 @@
+import { sign_in_selectors as sign_in_selectors } from "../selectors/sign_in_selectors";
+
+describe('UI tests for sign in page', () => {
+        beforeEach('visiting sign in page', () => {
+            cy.visit('/');
+        })
+
+        it('1.should show typeable Username field"', () => {
+            cy.get(sign_in_selectors.username_field)
+                .should('be.visible')
+                .type('Hello World!')
+                .clear();
+        })
+
+        it('2.should show typeable Password field', () => {
+            cy.get(sign_in_selectors.password_field)
+                .should('be.visible')
+                .type('Hello World!')
+                .clear();
+        })
+
+        it('3.1 should show Username placeholder', () => {
+            cy.get(sign_in_selectors.username_lable)
+                .should('have.text', `Username`);
+        })
+
+        it('3.2 should show Password placeholder', () => {
+            cy.get(sign_in_selectors.password_lable)
+                .should('have.text', `Password`);
+        })
+
+        it('4.1 Error message for missed user name', () => {
+            cy.get(sign_in_selectors.password_field).click();
+            cy.get(sign_in_selectors.username_helper_text)
+                .should('be.visible')
+                .and('contain', 'Username is required')
+                .and('have.css', 'color', 'rgb(244, 67, 54)');
+        })
+
+        it('4.2 Error message not appears', () => {
+            cy.get(sign_in_selectors.username_field).type('Hello World!');
+            cy.get(sign_in_selectors.password_field).click();
+            cy.get(sign_in_selectors.username_helper_text).should('not.exist');
+        })
+
+        it('5.1 Remember me disabled by default', () => {
```

Comment:
in test suite should be only tests which are listed here:
https://github.com/anatoly-devico/cypress_devico_courses/blob/main/cypress/e2e/authentication.spec.js